.circleci/config.yml

```diff
@@ -1,17 +1,27 @@
 version: 2
+
 jobs:
   build:
     working_directory: ~/code
     docker:
       - image: circleci/android:api-28
     steps:
       - checkout
+      - restore_cache:
```

Comment:
I assume changes in this file were needed but I'm not really following what they. Would you describe why you added them? PR description/Jira ticket didn't mention anything.

Reply:
The CI was failing, we were using an old and incorrect config. This is just basic CI config for Android. Cache will make CI builds faster since the same dependecies will not be pulled again and again.